pkg/cmd/cli/cmd/logs.go

```diff
@@ -63,6 +69,7 @@ func NewCmdLogs(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Com
 		}
 		cmdutil.CheckErr(o.RunLog())
 	}
+	cmd.Flags().String("version", "", "View the logs of the deployment associated with the provided version for a deployment config")
```

Comment:
Also what should happen if someone hit `oc logs build/ruby-1 --version=2`? I guess we should error out?